uefi-firmware.nix

```diff
@@ -48,11 +54,14 @@ let
     sha256 = "sha256-EPtI63jYhEIo4uVTH3lUt9NC/lK5vPVacUAc5qgmz9M=";
   };
 
-  _edk2-nvidia = fetchFromGitHub {
-    owner = "NVIDIA";
-    repo = "edk2-nvidia";
-    rev = "ad78b07af65d41bb96839f0bbe67bb445e04272f"; # Latest on r35.3.1-updates as of 2023-05-01
-    sha256 = "sha256-PdrisHYkmBXGvfkNboVvKJnBqORiM8sUsGySj7n5Y5c=";
+  _edk2-nvidia = applyPatches {
+    src = fetchFromGitHub {
+      owner = "NVIDIA";
+      repo = "edk2-nvidia";
+      rev = "ad78b07af65d41bb96839f0bbe67bb445e04272f"; # Latest on r35.3.1-updates as of 2023-05-01
+      sha256 = "sha256-PdrisHYkmBXGvfkNboVvKJnBqORiM8sUsGySj7n5Y5c=";
+    };
+    patches = [ ./capsule-authentication.patch ];
   };
   edk2-nvidia =
     if (errorLevelInfo || bootLogo != null)
```

Comment:
This might be a bug I introduced. I think we'd need to conditionally use the applyPatches version if edk2NvidiaPatches is nonempty.

Reply:
Should be fixed now